package.json

```diff
@@ -78,6 +78,7 @@
     ]
   },
   "dependencies": {
+    "omnitone": "^1.3.0",
```

Comment:
I think what @gkatsev meant was that this should only be a dev dependency. Then we have the `omniture` option be the actual `Omniture` library.